src/com/trollworks/gcs/feature/SkillBonus.java

```diff
@@ -98,28 +103,41 @@ public String getKey() {
         return buffer.toString();
     }
 
+    public boolean matchesCategories(String categories) {
+        String[] cats = categories.split(COMMA);
+        for (String category : cats) {
+            if (mCategoryCriteria.matches(category.trim())) {
```

Comment:
Should probably do a case-insensitive comparison

Reply:
Actually, it is already case insensitive.   The StringCriteria code handles that.